src/Valet/Commands/Forecast.cs

```diff
@@ -28,18 +28,22 @@ public Forecast(string[] args)
     private static readonly Option<FileInfo[]> SourceFilePath = new("--source-file-path")
     {
         Description = "The file path(s) to existing jobs data.",
-        IsRequired = false,
+        IsRequired = true,
         AllowMultipleArgumentsPerToken = true
     };
 
+    protected override List<Option> Options => new()
+    {
+        SourceFilePath
+    };
+
     protected override Command GenerateCommand(App app)
     {
         var command = base.GenerateCommand(app);
         command.AppendGeneralRequiredOptions();
 
         command.AddGlobalOption(StartDate);
         command.AddGlobalOption(TimeSlice);
-        command.AddGlobalOption(SourceFilePath);
 
```

Comment:
What's the difference between adding as a global option, and doing it this way?

Reply:
Defining it as a global option would make `--source-file-path` a required option for all subcommands. The way it's written now, `--source-file-path` is required for the base forecast command (provider-agnostic), and optional when forecast is called with a subcommand (`forecast jenkins ...`)